xml/operations-opsconsole-en-alarm_explorer.xml

```diff
@@ -10,89 +10,25 @@
  <title>Alarm Explorer</title>
  <para>
   This page displays the alarms for all services and appliances. By default,
-  alarms are sorted by most recent and in a table format.
+  alarms are sorted by their state.
  </para>
  <section>
-  <title>Search and Sort</title>
+  <title>Filter and Sort</title>
   <para>
-   The search bar allows you to filter the alarms you want to view. Type and
-   Status are examples of alarm criteria that can be specified. Additionally,
-   you can filter by typing in text similar to searching by keywords.
+   Using the <guimenu>Filter Alarms</guimenu> button, you can filter the
+   alarms by their IDs and dimensions. The <guimenu>Filter Alarms</guimenu>
+   dialog lets you configure a filtering rule using the <guimenu>Alarm
+   ID</guimenu> field and options in the <guimenu>Dimension(s)</guimenu>
+   section.
   </para>
-  <para>
-   The checkbox allows you to select (or deselect) a group of alarms and apply
-   a single condition on the selected alarms:
-  </para>
-  <itemizedlist>
-   <listitem>
-    <para>
-     <guimenu>Select Visible</guimenu> allows you to set a condition on
-     selected alarms from the table.
-    </para>
-   </listitem>
-   <listitem>
-    <para>
-     <guimenu>Select All</guimenu> allows you to set a condition on all the
-     alarms from the table.
-    </para>
-   </listitem>
-   <listitem>
-    <para>
-     <guimenu>Clear Selection</guimenu> allows you to clear all the selections
-     currently selected from the table.
-    </para>
-   </listitem>
-  </itemizedlist>
   <para>
    You can display the alarms by grid, list or table views by selecting the
    corresponding icons next to the <guimenu>Sort By</guimenu> control.
   </para>
   <para>
-   The <guimenu>Sort By</guimenu> control contains a drop-down list of ways by
-   which you can sort the alarms. Alternatively, you can also sort using the
-   column headers in the table.
+   To sort the alarm list, click on the desired column header. To reverse the
+   sort order click again on the column.
```

Comment:
Sunds a bit clumsy near the end, suggestion: "...order, click the column again."